Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,62 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+joint operational direction of the Node.js Technical Steering Committee (TSC)
+and Node.js Foundation Community Committee (CommComm).
```

Comment:
Since we're listing out owners, I believe this should be "Node.js Technical Steering Committee (TSC), Node.js Foundation Community Committee (CommComm), and the Node.js Foundation Board," since there are a few repos (nodejs/board, nodejs/live.nodejs.org, others?) that are directly operated by the board.